support/cas-server-support-gauth/src/main/java/org/apereo/cas/config/support/authentication/GoogleAuthenticatorAuthenticationEventExecutionPlanConfiguration.java

```diff
@@ -194,10 +196,16 @@ public CipherExecutor googleAuthenticatorAccountCipherExecutor() {
         return CipherExecutor.noOp();
     }
 
+    @Bean
+    public OneTimeTokenAccountValidator googleAuthenticatorAccountValidator() {
```

Comment:
Adding missing annotations for refresh-scope and condition-on-missing-bean

Reply:
639eba8179badc3b14ca8966cc87c557b3f75167